src/shared/Elements/H1.tsx

```diff
@@ -15,12 +15,13 @@ interface IH1Props {
 }
 
 export const H1 = styled.h1<IH1Props>`
-  font-size: ${(props) => props.fontSize || '36px'};
+  font-size: ${(props) => props.fontSize || '32px'};
+  font-faimly: ${(props) => props.theme.fonts.header};
   text-align: ${(props) => props.textAlign || 'left'};
   color: ${(props) => props.color || props.theme.colors.red};
-  margin-left: ${(props) => props.marginLeft || '18px'};
-  margin-bottom: ${(props) => props.marginBottom || '18px'};
-  margin-top: ${(props) => props.marginTop || 'initial'};
+  margin-left: ${(props) => props.marginLeft || '0'};
```

Comment:
Have you confirmed that these style changes do not break other parts of the website?

Reply:
It probably does mess with styling on other pages. We'll be redoing every page tho, and think 0 horizontal margin is a more standard default